File: alexandria/ui/src/io/intino/alexandria/ui/displays/Root.java
package io.intino.alexandria.ui.displays;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.RootNotifier;

public class Root<DN extends RootNotifier, B extends Box> extends Display<DN, B> {

	public Root(B box) {
		super(box);
	}

}

File: alexandria/ui-elements/src/displays/Root.js
import React from "react";
import AbstractRoot from "../../gen/displays/AbstractRoot";
import RootNotifier from "../../gen/displays/notifiers/RootNotifier";
import RootRequester from "../../gen/displays/requesters/RootRequester";

export default class Root extends AbstractRoot {

	constructor(props) {
		super(props);
		this.notifier = new RootNotifier(this);
		this.requester = new RootRequester(this);
	};

	render() {
		return (
			<React.Fragment>
			</React.Fragment>
		);
	};


}